develop/tutorials/articles/111-customizing/06-overriding-portal-properties.markdown

```diff
@@ -1,16 +1,16 @@
 # Overriding Portal Properties using a Hook [](id=overriding-portal-properties-using-a-hook)
 
 A portal properties hook plugin lets you override a subset of
-[portal properties](@platform-ref@/7.0/propertiesdoc/portal.properties.html)
+[portal properties](@platform-ref@/7.0-ga1/propertiesdoc/portal.properties.html)
```

Comment:
@justinchoi001 Can you change it to `7.0-latest`? That's our "evergreen" symbolic link. Thanks for catching this.